README.md

```diff
@@ -30,19 +30,21 @@ For a quick introduction to Joinmarket you can watch [this demonstration](https:
 
 ### Quickstart - RECOMMENDED INSTALLATION METHOD (Linux and macOS only)
 
-Once you've downloaded this repo, either as a tar/zip file, and extracted it, or via `git clone`:
+Download the latest [release](https://github.com/Joinmarket-Org/joinmarket-clientserver/releases) as tar or zip, and extract it.
 
-Make sure to validate the signature on the tar/zip file provided on the [release page](https://github.com/Joinmarket-Org/joinmarket-clientserver/releases),
-or check the signature in git if you install that way using `git log --show-signature`.
+Make sure to validate the signature on the tar/zip file provided with the [release](https://github.com/Joinmarket-Org/joinmarket-clientserver/releases) (
+or check the signature in git if you install that way using `git log --show-signature`).
 
 **macOS users**: Make sure that you have Homebrew and Apple's Command Line Tools installed.
 
     ./install.sh
-    (follow instructions on screen; provide sudo password when prompted)
+
+(follow instructions on screen; provide sudo password when prompted)
+
     source jmvenv/bin/activate
     cd scripts
 
-(You can add `--develop` as an extra flag to `install.sh` to make the Joinmarket code editable in-place.)
+(You can add `--develop` as an extra flag to `install.sh` to make the Joinmarket code editable in-place. See more options with `./install.sh -?`.)
```

Comment:
Should we keep `--develop`? It's no-op for some time already...

Reply:
I mean - keep in README.

Reply:
Oh yes good point. Will remove.

Reply:
Rewrote it a bit to: move the install.sh option instruction to the right place, and also () for the macOS note, because otherwise the reader isn't sure whether the next text is also macOS specific.